numba/cuda/tests/doc_examples/test_ffi.py

```diff
@@ -45,6 +45,38 @@ def multiply_vectors(r, x, y):
         np.testing.assert_array_equal(r, x * y)
         # magictoken.ex_linking_cu.end
 
+    def test_ex_from_buffer(self):
```

Comment:
Does this test need some `@skip` for environments in which it is not designed to work?

Reply:
It's already skipped on cudasim and in environments where the NVIDIA bindings aren't available - the CC doesn't matter because it's linked with C++ source that is compiled for whatever the current CC is.